### PR TITLE
KM document slideseq test data

### PR DESCRIPTION
**Pull request by kayleemathews**

### CODE REVIEW

**Review by ekiernan — CHANGES_REQUESTED** [review]
I suggest adding a few minor changes to the cropping description just to clarify for folks.

**Review by khajoue2 — APPROVED** [review]
LGTM

File: pipelines/skylab/slideseq/test_inputs/test_data_overview.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+The following lists the data used for workflow validation, including workflow engineering (plumbing) tests and scientific validation tests.
+
+All unmodified files are stored in a public Google bucket (gs://nemo-public/biccn-unbundled/grant/rf1_macosko/macosko/spatial_transcriptome/cellgroup/Slide-seq/mouse/raw/) and can be identified using their sample ID (ex. Puck_210817_11).
+
+## Plumbing
+
+* Puck_210817_11.human
+
+The plumbing input listed above was downsampled using a [custom python script](Plumbing/slideseq_downsample.py) to first crop the sample from 2500 to 4000 in the x-dimension and 1000 to 4200 in the y-dimension. Next, 10 random cells were sampled for each 30 by 30 region in the cropped sample. The list of sampled cells was then used to create new, downsampled FASTQ files using a second [custom script](https://github.com/broadinstitute/warp-tools/blob/master/fastqpreprocessing/src/samplefastq.cpp) from the [warp-tools GitHub repository](https://github.com/broadinstitute/warp-tools/tree/master).
```
Comment (ekiernan) [inline pipelines/skylab/slideseq/test_inputs/test_data_overview.md:11]:
```suggestion
The plumbing input FASTQ listed above was downsampled using a [custom python script](Plumbing/slideseq_downsample.py). First, a PDF image of the sample puck was examined and the XY coordinates for a region of interest were selected. A bead position TSV containing the XY coordinates for the puck's barcodes was used as input to the custom script and the TSV was cropped 2500 to 4000 in the x-dimension and 1000 to 4200 in the y-dimension. Next, 10 random cells were sampled for each 30 by 30 region in the cropped TSV. The list of sampled cells was then used to create new, downsampled FASTQ files using a second [custom script](https://github.com/broadinstitute/warp-tools/blob/master/fastqpreprocessing/src/samplefastq.cpp) from the [warp-tools GitHub repository](https://github.com/broadinstitute/warp-tools/tree/master). 
```

File: pipelines/skylab/slideseq/test_inputs/test_data_overview.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# Slideseq Test Data
+
+The following lists the data used for workflow validation, including workflow engineering (plumbing) tests and scientific validation tests.
+
+All unmodified files are stored in a public Google bucket (gs://nemo-public/biccn-unbundled/grant/rf1_macosko/macosko/spatial_transcriptome/cellgroup/Slide-seq/mouse/raw/) and can be identified using their sample ID (ex. Puck_210817_11).
+
+## Plumbing
+
+* Puck_210817_11.human
```
[the remark this answers is not in the document]
Comment (kayleemathews) [reply pipelines/skylab/slideseq/test_inputs/test_data_overview.md:9]: Do these live anywhere other than in the SlideSeq dev workspace bucket? Somewhere public we can link to? @khajoue2

File: pipelines/skylab/slideseq/test_inputs/test_data_overview.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kayleemathews) [reply pipelines/skylab/slideseq/test_inputs/test_data_overview.md]: Updated

**Pull request merged**